src/pages/resultsView/enrichments/CopyNumberEnrichmentsTab.tsx

```diff
@@ -37,18 +37,40 @@ export default class CopyNumberEnrichmentsTab extends React.Component<ICopyNumbe
-                                                       store={this.props.store} alterationType="a deep deletion"/>
+                            groups={[
+                                {
+                                    name: "Altered group",
+                                    description: "samples that have alterations in the query gene(s) that also have a deep deletion in the listed gene.",
+                                    displayName: "Co-occurrence",
+                                    count: this.props.store.alteredSampleKeys.result!.length
+                                }, {
+                                    name: "Unaltered group",
+                                    description: "samples that do not have alterations in the query gene(s) that have a deep deletion in the listed gene.",
+                                    displayName: "Mutual exclusivity",
+                                    count: this.props.store.unalteredSampleKeys.result!.length
+                                }
+                            ]}
+                            selectedProfile={this.props.store.selectedEnrichmentCopyNumberProfile}
+                            headerName={"Deep Deletion - " + this.props.store.selectedEnrichmentCopyNumberProfile.name}
+                            store={this.props.store} />
                         <hr />
                         <AlterationEnrichmentContainer data={this.props.store.copyNumberAmpEnrichmentData.result!}
-                                                       totalGroup1Count={this.props.store.alteredSampleKeys.result!.length}
-                                                       totalGroup2Count={this.props.store.unalteredSampleKeys.result!.length}
-                                                       selectedProfile={this.props.store.selectedEnrichmentCopyNumberProfile}
-                                                       headerName={"Amplification - " + this.props.store.selectedEnrichmentCopyNumberProfile.name}
-                                                       store={this.props.store} alterationType="an amplification"/>
+                            groups={[
+                                {
+                                    name: "Altered group",
+                                    description: "samples that have alterations in the query gene(s) that also have an amplification in the listed gene.",
+                                    displayName: "Co-occurrence",
+                                    count: this.props.store.alteredSampleKeys.result!.length
+                                }, {
+                                    name: "Unaltered group",
+                                    description: "samples that do not have alterations in the query gene(s) that have an amplification in the listed gene.",
+                                    displayName: "Mutual exclusivity",
```

Comment:
this is very confusing. I understand whats going on but it's confusing. I think the best way to proceed is to rename `displayName`. Maybe it should be `directionNameInPValuePlot` or something like that